[2025-02-17T14:06:34.923+0000] {processor.py:157} INFO - Started process (PID=170) to work on /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:06:34.925+0000] {processor.py:829} INFO - Processing file /opt/airflow/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:06:34.927+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:06:34.926+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:06:37.112+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:06:37.100+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:06:37.116+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:06:37.140+0000] {processor.py:179} INFO - Processing /opt/airflow/airflow/dags/etl_pipeline.py took 2.221 seconds
[2025-02-17T14:07:07.954+0000] {processor.py:157} INFO - Started process (PID=190) to work on /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:07:07.955+0000] {processor.py:829} INFO - Processing file /opt/airflow/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:07:07.957+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:07:07.956+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:07:08.625+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:07:08.613+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:07:08.629+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:07:08.673+0000] {processor.py:179} INFO - Processing /opt/airflow/airflow/dags/etl_pipeline.py took 0.724 seconds
[2025-02-17T14:07:39.058+0000] {processor.py:157} INFO - Started process (PID=196) to work on /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:07:39.060+0000] {processor.py:829} INFO - Processing file /opt/airflow/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:07:39.061+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:07:39.061+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:07:39.875+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:07:39.860+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:07:39.880+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:07:39.901+0000] {processor.py:179} INFO - Processing /opt/airflow/airflow/dags/etl_pipeline.py took 0.849 seconds
[2025-02-17T14:08:10.635+0000] {processor.py:157} INFO - Started process (PID=202) to work on /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:08:10.636+0000] {processor.py:829} INFO - Processing file /opt/airflow/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:08:10.637+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:08:10.637+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:08:11.285+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:08:11.270+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:08:11.289+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:08:11.335+0000] {processor.py:179} INFO - Processing /opt/airflow/airflow/dags/etl_pipeline.py took 0.705 seconds
[2025-02-17T14:08:41.835+0000] {processor.py:157} INFO - Started process (PID=208) to work on /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:08:41.837+0000] {processor.py:829} INFO - Processing file /opt/airflow/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:08:41.838+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:08:41.838+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:08:42.495+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:08:42.484+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:08:42.499+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:08:42.521+0000] {processor.py:179} INFO - Processing /opt/airflow/airflow/dags/etl_pipeline.py took 0.689 seconds
[2025-02-17T14:09:13.232+0000] {processor.py:157} INFO - Started process (PID=214) to work on /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:09:13.234+0000] {processor.py:829} INFO - Processing file /opt/airflow/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:09:13.236+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:09:13.235+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:09:13.868+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:09:13.857+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:09:13.873+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:09:13.893+0000] {processor.py:179} INFO - Processing /opt/airflow/airflow/dags/etl_pipeline.py took 0.664 seconds
[2025-02-17T14:09:44.472+0000] {processor.py:157} INFO - Started process (PID=220) to work on /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:09:44.473+0000] {processor.py:829} INFO - Processing file /opt/airflow/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:09:44.475+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:09:44.474+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:09:45.087+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:09:45.071+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:09:45.093+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:09:45.115+0000] {processor.py:179} INFO - Processing /opt/airflow/airflow/dags/etl_pipeline.py took 0.648 seconds
[2025-02-17T14:10:15.825+0000] {processor.py:157} INFO - Started process (PID=226) to work on /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:10:15.827+0000] {processor.py:829} INFO - Processing file /opt/airflow/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:10:15.828+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:10:15.827+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:10:16.442+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:10:16.429+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:10:16.447+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:10:16.493+0000] {processor.py:179} INFO - Processing /opt/airflow/airflow/dags/etl_pipeline.py took 0.672 seconds
[2025-02-17T14:10:47.109+0000] {processor.py:157} INFO - Started process (PID=232) to work on /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:10:47.110+0000] {processor.py:829} INFO - Processing file /opt/airflow/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:10:47.111+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:10:47.111+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:10:47.704+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:10:47.691+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:10:47.707+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/airflow/dags/etl_pipeline.py
[2025-02-17T14:10:47.756+0000] {processor.py:179} INFO - Processing /opt/airflow/airflow/dags/etl_pipeline.py took 0.651 seconds
[2025-02-17T14:14:32.755+0000] {processor.py:157} INFO - Started process (PID=171) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:14:32.756+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:14:32.758+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:14:32.757+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:14:34.371+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:14:34.358+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:14:34.376+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:14:34.395+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.645 seconds
[2025-02-17T14:15:04.766+0000] {processor.py:157} INFO - Started process (PID=191) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:15:04.768+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:15:04.771+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:15:04.770+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:15:05.797+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:15:05.782+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:15:05.801+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:15:05.825+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.066 seconds
[2025-02-17T14:15:36.792+0000] {processor.py:157} INFO - Started process (PID=197) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:15:36.793+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:15:36.795+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:15:36.794+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:15:37.616+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:15:37.603+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:15:37.620+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:15:37.672+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.884 seconds
[2025-02-17T14:16:07.752+0000] {processor.py:157} INFO - Started process (PID=218) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:16:07.753+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:16:07.755+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:16:07.754+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:16:08.414+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:16:08.400+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:16:08.419+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:16:08.441+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.694 seconds
[2025-02-17T14:16:38.696+0000] {processor.py:157} INFO - Started process (PID=224) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:16:38.697+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:16:38.698+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:16:38.698+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:16:39.326+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:16:39.315+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:16:39.330+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:16:39.376+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.684 seconds
[2025-02-17T14:17:09.472+0000] {processor.py:157} INFO - Started process (PID=230) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:17:09.474+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:17:09.475+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:17:09.475+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:17:10.100+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:17:10.088+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:17:10.104+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:17:10.150+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.682 seconds
[2025-02-17T14:17:40.286+0000] {processor.py:157} INFO - Started process (PID=243) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:17:40.288+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:17:40.290+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:17:40.289+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:17:40.984+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:17:40.971+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:17:40.988+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:17:41.009+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.729 seconds
[2025-02-17T14:18:17.692+0000] {processor.py:157} INFO - Started process (PID=172) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:18:17.693+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:18:17.694+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:18:17.694+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:18:19.938+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:18:19.923+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:18:19.943+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:18:19.968+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.635 seconds
[2025-02-17T14:18:50.333+0000] {processor.py:157} INFO - Started process (PID=181) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:18:50.336+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:18:50.337+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:18:50.337+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:18:50.907+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:18:50.895+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:18:50.911+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:18:50.955+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.625 seconds
[2025-02-17T14:19:21.005+0000] {processor.py:157} INFO - Started process (PID=190) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:19:21.006+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:19:21.007+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:19:21.007+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:19:21.613+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:19:21.603+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:19:21.616+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:19:21.637+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.636 seconds
[2025-02-17T14:19:51.954+0000] {processor.py:157} INFO - Started process (PID=198) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:19:51.956+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:19:51.957+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:19:51.957+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:19:52.656+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:19:52.642+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:19:52.661+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:19:52.720+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.769 seconds
[2025-02-17T14:20:22.815+0000] {processor.py:157} INFO - Started process (PID=216) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:20:22.817+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:20:22.819+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:20:22.818+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:20:23.569+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:20:23.556+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:20:23.573+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:20:23.596+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.788 seconds
[2025-02-17T14:20:53.673+0000] {processor.py:157} INFO - Started process (PID=228) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:20:53.675+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:20:53.677+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:20:53.677+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:20:54.560+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:20:54.548+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:20:54.563+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:20:54.613+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.948 seconds
[2025-02-17T14:21:24.925+0000] {processor.py:157} INFO - Started process (PID=236) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:21:24.927+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:21:24.928+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:21:24.928+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:21:25.579+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:21:25.566+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:21:25.584+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:21:25.604+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.685 seconds
[2025-02-17T14:21:56.498+0000] {processor.py:157} INFO - Started process (PID=243) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:21:56.500+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:21:56.501+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:21:56.501+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:21:57.114+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:21:57.099+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:21:57.119+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:21:57.164+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.670 seconds
[2025-02-17T14:22:27.652+0000] {processor.py:157} INFO - Started process (PID=249) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:22:27.653+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:22:27.654+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:22:27.654+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:22:28.292+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:22:28.279+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:22:28.297+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:22:28.317+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.670 seconds
[2025-02-17T14:22:58.420+0000] {processor.py:157} INFO - Started process (PID=257) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:22:58.421+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:22:58.423+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:22:58.422+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:22:59.045+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:22:59.032+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:22:59.051+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:22:59.076+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.661 seconds
[2025-02-17T14:23:29.789+0000] {processor.py:157} INFO - Started process (PID=276) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:23:29.791+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:23:29.792+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:23:29.792+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:23:31.464+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:23:31.634+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:23:31.634+0000] {manager.py:499} INFO - Created Permission View: %s
[2025-02-17T14:23:31.650+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:23:31.650+0000] {manager.py:499} INFO - Created Permission View: %s
[2025-02-17T14:23:31.661+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:23:31.661+0000] {manager.py:499} INFO - Created Permission View: %s
[2025-02-17T14:23:31.663+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:23:31.662+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:23:31.680+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:23:31.679+0000] {dag.py:2963} INFO - Creating ORM DAG for ETL_Pipeline
[2025-02-17T14:23:31.695+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:23:31.695+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-16T00:00:00+00:00, run_after=2025-02-17T00:00:00+00:00
[2025-02-17T14:23:31.724+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.939 seconds
[2025-02-17T14:24:01.812+0000] {processor.py:157} INFO - Started process (PID=287) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:24:01.814+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:24:01.815+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:24:01.815+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:24:02.916+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:24:02.945+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:24:02.945+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:24:02.975+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:24:02.974+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-16T00:00:00+00:00, run_after=2025-02-17T00:00:00+00:00
[2025-02-17T14:24:03.002+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.196 seconds
[2025-02-17T14:24:33.164+0000] {processor.py:157} INFO - Started process (PID=293) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:24:33.166+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:24:33.167+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:24:33.167+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:24:34.189+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:24:34.217+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:24:34.217+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:24:34.243+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:24:34.243+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-16T00:00:00+00:00, run_after=2025-02-17T00:00:00+00:00
[2025-02-17T14:24:34.293+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.133 seconds
[2025-02-17T14:25:04.442+0000] {processor.py:157} INFO - Started process (PID=335) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:25:04.444+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:25:04.446+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:25:04.445+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:25:05.361+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:25:05.391+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:25:05.390+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:25:05.417+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:25:05.416+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:25:05.442+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.004 seconds
[2025-02-17T14:25:36.429+0000] {processor.py:157} INFO - Started process (PID=341) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:25:36.432+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:25:36.435+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:25:36.434+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:25:37.943+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:25:37.984+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:25:37.983+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:25:38.017+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:25:38.017+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:25:38.073+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.651 seconds
[2025-02-17T14:26:08.443+0000] {processor.py:157} INFO - Started process (PID=347) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:26:08.444+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:26:08.448+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:26:08.448+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:26:09.808+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:26:09.845+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:26:09.844+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:26:09.927+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:26:09.926+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:26:09.956+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.517 seconds
[2025-02-17T14:26:40.089+0000] {processor.py:157} INFO - Started process (PID=353) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:26:40.090+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:26:40.092+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:26:40.091+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:26:41.031+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:26:41.057+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:26:41.057+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:26:41.083+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:26:41.083+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:26:41.107+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.021 seconds
[2025-02-17T14:27:11.191+0000] {processor.py:157} INFO - Started process (PID=376) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:27:11.193+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:27:11.195+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:27:11.195+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:27:12.420+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:27:12.453+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:27:12.453+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:27:12.483+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:27:12.483+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:27:12.511+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.324 seconds
[2025-02-17T14:27:43.266+0000] {processor.py:157} INFO - Started process (PID=382) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:27:43.268+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:27:43.269+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:27:43.269+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:27:44.279+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:27:44.311+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:27:44.311+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:27:44.345+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:27:44.345+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:27:44.373+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.111 seconds
[2025-02-17T14:28:15.367+0000] {processor.py:157} INFO - Started process (PID=388) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:28:15.368+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:28:15.369+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:28:15.369+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:28:16.309+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:28:16.338+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:28:16.337+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:28:16.367+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:28:16.367+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:28:16.392+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.029 seconds
[2025-02-17T14:28:46.598+0000] {processor.py:157} INFO - Started process (PID=394) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:28:46.599+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:28:46.600+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:28:46.600+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:28:47.524+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:28:47.552+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:28:47.551+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:28:47.578+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:28:47.578+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:28:47.603+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.008 seconds
[2025-02-17T14:29:17.666+0000] {processor.py:157} INFO - Started process (PID=400) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:29:17.667+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:29:17.669+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:29:17.668+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:29:18.576+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:29:18.602+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:29:18.601+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:29:18.627+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:29:18.627+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:29:18.651+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.988 seconds
[2025-02-17T14:29:49.616+0000] {processor.py:157} INFO - Started process (PID=408) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:29:49.617+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:29:49.618+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:29:49.618+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:29:50.843+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:29:50.889+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:29:50.888+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:29:50.931+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:29:50.930+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:29:50.956+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.343 seconds
[2025-02-17T14:30:21.062+0000] {processor.py:157} INFO - Started process (PID=442) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:30:21.063+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:30:21.065+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:30:21.064+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:30:22.065+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:30:22.094+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:30:22.093+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:30:22.119+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:30:22.119+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:30:22.146+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.090 seconds
[2025-02-17T14:30:52.257+0000] {processor.py:157} INFO - Started process (PID=448) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:30:52.258+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:30:52.259+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:30:52.259+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:30:53.186+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:30:53.214+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:30:53.214+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:30:53.243+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:30:53.242+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:30:53.267+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.015 seconds
[2025-02-17T14:31:23.348+0000] {processor.py:157} INFO - Started process (PID=454) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:31:23.349+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:31:23.351+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:31:23.350+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:31:24.242+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:31:24.269+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:31:24.268+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:31:24.296+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:31:24.296+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:31:24.348+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.003 seconds
[2025-02-17T14:31:55.379+0000] {processor.py:157} INFO - Started process (PID=460) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:31:55.380+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:31:55.382+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:31:55.381+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:31:56.433+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:31:56.466+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:31:56.465+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:31:56.505+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:31:56.504+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:31:56.534+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.158 seconds
[2025-02-17T14:32:27.415+0000] {processor.py:157} INFO - Started process (PID=480) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:32:27.416+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:32:27.418+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:32:27.418+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:32:28.446+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:32:28.474+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:32:28.474+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:32:28.500+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:32:28.500+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:32:28.528+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.119 seconds
[2025-02-17T14:32:58.621+0000] {processor.py:157} INFO - Started process (PID=486) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:32:58.622+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:32:58.624+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:32:58.623+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:32:59.518+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:32:59.545+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:32:59.544+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:32:59.572+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:32:59.572+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:32:59.596+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.979 seconds
[2025-02-17T14:33:29.853+0000] {processor.py:157} INFO - Started process (PID=492) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:33:29.854+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:33:29.856+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:33:29.855+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:33:30.738+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:33:30.764+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:33:30.764+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:33:30.792+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:33:30.791+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:33:30.842+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.993 seconds
[2025-02-17T14:34:00.959+0000] {processor.py:157} INFO - Started process (PID=498) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:34:00.960+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:34:00.962+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:34:00.961+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:34:01.960+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:34:01.995+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:34:01.995+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:34:02.027+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:34:02.027+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:34:02.079+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.124 seconds
[2025-02-17T14:34:33.028+0000] {processor.py:157} INFO - Started process (PID=504) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:34:33.029+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:34:33.032+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:34:33.031+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:34:33.957+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:34:33.984+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:34:33.983+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:34:34.010+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:34:34.009+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:34:34.067+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.043 seconds
[2025-02-17T14:35:04.171+0000] {processor.py:157} INFO - Started process (PID=510) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:35:04.173+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:35:04.174+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:35:04.174+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:35:05.079+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:35:05.107+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:35:05.107+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:35:05.134+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:35:05.134+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:35:05.159+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.992 seconds
[2025-02-17T14:35:35.323+0000] {processor.py:157} INFO - Started process (PID=516) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:35:35.324+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:35:35.326+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:35:35.325+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:35:36.223+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:35:36.251+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:35:36.250+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:35:36.276+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:35:36.276+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:35:36.301+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.982 seconds
[2025-02-17T14:36:06.418+0000] {processor.py:157} INFO - Started process (PID=522) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:36:06.419+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:36:06.421+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:36:06.421+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:36:07.321+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:36:07.347+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:36:07.346+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:36:07.371+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:36:07.371+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:36:07.394+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.980 seconds
[2025-02-17T14:36:37.466+0000] {processor.py:157} INFO - Started process (PID=528) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:36:37.468+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:36:37.469+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:36:37.469+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:36:38.380+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:36:38.406+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:36:38.406+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:36:38.433+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:36:38.432+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:36:38.482+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.020 seconds
[2025-02-17T14:37:08.551+0000] {processor.py:157} INFO - Started process (PID=534) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:37:08.553+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:37:08.555+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:37:08.554+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:37:09.412+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:37:09.438+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:37:09.438+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:37:09.463+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:37:09.463+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:37:09.489+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.941 seconds
[2025-02-17T14:37:39.615+0000] {processor.py:157} INFO - Started process (PID=540) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:37:39.616+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:37:39.617+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:37:39.617+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:37:40.617+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:37:40.651+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:37:40.650+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:37:40.678+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:37:40.677+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:37:40.712+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.101 seconds
[2025-02-17T14:38:10.786+0000] {processor.py:157} INFO - Started process (PID=546) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:38:10.788+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:38:10.789+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:38:10.789+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:38:11.964+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:38:12.001+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:38:12.001+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:38:12.036+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:38:12.035+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:38:12.070+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.288 seconds
[2025-02-17T14:38:42.126+0000] {processor.py:157} INFO - Started process (PID=552) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:38:42.127+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:38:42.128+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:38:42.128+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:38:43.128+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:38:43.154+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:38:43.153+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:38:43.181+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:38:43.181+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:38:43.203+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.082 seconds
[2025-02-17T14:39:13.333+0000] {processor.py:157} INFO - Started process (PID=558) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:39:13.334+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:39:13.336+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:39:13.335+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:39:14.391+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:39:14.424+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:39:14.423+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:39:14.460+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:39:14.460+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:39:14.516+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.188 seconds
[2025-02-17T14:39:44.668+0000] {processor.py:157} INFO - Started process (PID=564) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:39:44.669+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:39:44.671+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:39:44.670+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:39:45.639+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:39:45.665+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:39:45.665+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:39:45.689+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:39:45.689+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:39:45.738+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.073 seconds
[2025-02-17T14:40:16.702+0000] {processor.py:157} INFO - Started process (PID=570) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:40:16.704+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:40:16.705+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:40:16.704+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:40:17.614+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:40:17.642+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:40:17.641+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:40:17.667+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:40:17.667+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:40:17.723+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.033 seconds
[2025-02-17T14:40:47.824+0000] {processor.py:157} INFO - Started process (PID=576) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:40:47.826+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:40:47.827+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:40:47.827+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:40:48.765+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:40:48.794+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:40:48.794+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:40:48.824+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:40:48.824+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:40:48.875+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.054 seconds
[2025-02-17T14:46:32.186+0000] {processor.py:157} INFO - Started process (PID=171) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:46:32.187+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:46:32.189+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:46:32.188+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:46:33.689+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:46:33.679+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:46:33.693+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:46:33.710+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.528 seconds
[2025-02-17T14:47:04.264+0000] {processor.py:157} INFO - Started process (PID=183) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:47:04.265+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:47:04.267+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:47:04.266+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:47:04.975+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:47:04.964+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:47:04.978+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:47:05.001+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.742 seconds
[2025-02-17T14:47:35.098+0000] {processor.py:157} INFO - Started process (PID=191) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:47:35.099+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:47:35.100+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:47:35.100+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:47:35.752+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:47:35.743+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:47:35.756+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:47:35.775+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.682 seconds
[2025-02-17T14:48:05.878+0000] {processor.py:157} INFO - Started process (PID=214) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:48:05.880+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:48:05.882+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:48:05.881+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:48:06.512+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:48:06.497+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:48:06.517+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:48:06.565+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.691 seconds
[2025-02-17T14:48:36.708+0000] {processor.py:157} INFO - Started process (PID=222) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:48:36.709+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:48:36.710+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:48:36.710+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:48:37.301+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:48:37.290+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:48:37.304+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:48:37.348+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.644 seconds
[2025-02-17T14:49:07.426+0000] {processor.py:157} INFO - Started process (PID=231) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:49:07.427+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:49:07.429+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:49:07.428+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:49:08.055+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:49:08.042+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:49:08.061+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:49:08.083+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.662 seconds
[2025-02-17T14:49:40.119+0000] {processor.py:157} INFO - Started process (PID=239) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:49:40.120+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:49:40.121+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:49:40.121+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:49:41.309+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:49:41.290+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T14:49:41.314+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:49:41.346+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.231 seconds
[2025-02-17T14:50:11.530+0000] {processor.py:157} INFO - Started process (PID=258) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:50:11.532+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:50:11.533+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:50:11.533+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:50:13.185+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:50:13.454+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:50:13.452+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:50:13.496+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:50:13.496+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:50:13.538+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 2.012 seconds
[2025-02-17T14:50:43.691+0000] {processor.py:157} INFO - Started process (PID=269) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:50:43.692+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:50:43.694+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:50:43.693+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:50:45.089+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:50:45.126+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:50:45.125+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:50:45.159+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:50:45.159+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:50:45.190+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.509 seconds
[2025-02-17T14:51:15.356+0000] {processor.py:157} INFO - Started process (PID=301) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:51:15.358+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:51:15.359+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:51:15.359+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:51:16.478+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:51:16.506+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:51:16.505+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:51:16.554+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:51:16.553+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:51:16.619+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.267 seconds
[2025-02-17T14:51:47.511+0000] {processor.py:157} INFO - Started process (PID=307) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:51:47.513+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:51:47.514+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:51:47.514+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:51:48.479+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:51:48.507+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:51:48.506+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:51:48.535+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:51:48.534+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:51:48.562+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.054 seconds
[2025-02-17T14:52:19.664+0000] {processor.py:157} INFO - Started process (PID=313) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:52:19.666+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:52:19.667+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:52:19.667+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:52:21.300+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:52:21.338+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:52:21.337+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:52:21.377+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:52:21.376+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:52:21.410+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.750 seconds
[2025-02-17T14:52:51.648+0000] {processor.py:157} INFO - Started process (PID=320) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:52:51.650+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:52:51.651+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:52:51.650+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:52:52.624+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:52:52.665+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:52:52.664+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:52:52.703+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:52:52.702+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:52:52.756+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.112 seconds
[2025-02-17T14:53:22.845+0000] {processor.py:157} INFO - Started process (PID=326) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:53:22.847+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:53:22.848+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:53:22.848+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:53:23.783+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:53:23.815+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:53:23.813+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:53:23.843+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:53:23.843+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:53:23.898+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.057 seconds
[2025-02-17T14:53:55.687+0000] {processor.py:157} INFO - Started process (PID=332) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:53:55.688+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:53:55.690+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:53:55.689+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:53:56.646+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:53:56.678+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:53:56.677+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:53:56.705+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:53:56.705+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:53:56.756+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.073 seconds
[2025-02-17T14:54:27.143+0000] {processor.py:157} INFO - Started process (PID=364) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:54:27.144+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:54:27.146+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:54:27.146+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:54:28.125+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:54:28.152+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:54:28.152+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:54:28.177+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:54:28.177+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:54:28.227+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.089 seconds
[2025-02-17T14:54:58.314+0000] {processor.py:157} INFO - Started process (PID=370) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:54:58.316+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:54:58.317+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:54:58.317+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:54:59.265+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:54:59.293+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:54:59.293+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:54:59.322+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:54:59.321+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:54:59.347+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.038 seconds
[2025-02-17T14:55:30.501+0000] {processor.py:157} INFO - Started process (PID=376) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:55:30.503+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:55:30.504+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:55:30.504+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:55:31.459+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:55:31.487+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:55:31.487+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:55:31.514+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:55:31.514+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:55:31.568+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.072 seconds
[2025-02-17T14:56:02.661+0000] {processor.py:157} INFO - Started process (PID=382) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:56:02.662+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:56:02.664+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:56:02.664+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:56:03.598+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:56:03.626+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:56:03.625+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:56:03.654+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:56:03.654+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:56:03.676+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.019 seconds
[2025-02-17T14:56:34.810+0000] {processor.py:157} INFO - Started process (PID=388) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:56:34.812+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:56:34.813+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:56:34.812+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:56:35.718+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:56:35.747+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:56:35.745+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:56:35.773+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:56:35.772+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:56:35.798+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.991 seconds
[2025-02-17T14:57:07.003+0000] {processor.py:157} INFO - Started process (PID=394) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:57:07.004+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:57:07.005+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:57:07.005+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:57:07.996+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:57:08.022+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:57:08.022+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:57:08.048+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:57:08.047+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:57:08.071+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.071 seconds
[2025-02-17T14:57:38.138+0000] {processor.py:157} INFO - Started process (PID=400) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:57:38.139+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:57:38.140+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:57:38.140+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:57:39.061+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:57:39.090+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:57:39.090+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:57:39.119+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:57:39.119+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:57:39.143+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.009 seconds
[2025-02-17T14:58:10.143+0000] {processor.py:157} INFO - Started process (PID=406) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:58:10.144+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:58:10.145+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:58:10.145+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:58:11.073+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:58:11.101+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:58:11.101+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:58:11.128+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:58:11.128+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:58:11.152+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.012 seconds
[2025-02-17T14:58:42.360+0000] {processor.py:157} INFO - Started process (PID=412) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:58:42.361+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:58:42.362+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:58:42.362+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:58:43.305+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:58:43.331+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:58:43.331+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:58:43.358+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:58:43.358+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:58:43.392+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.036 seconds
[2025-02-17T14:59:14.501+0000] {processor.py:157} INFO - Started process (PID=418) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:59:14.502+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:59:14.504+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:59:14.503+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:59:15.447+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:59:15.474+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:59:15.473+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:59:15.502+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:59:15.501+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:59:15.553+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.056 seconds
[2025-02-17T14:59:45.687+0000] {processor.py:157} INFO - Started process (PID=424) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:59:45.688+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T14:59:45.689+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:59:45.689+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:59:46.661+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T14:59:46.688+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:59:46.688+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T14:59:46.715+0000] {logging_mixin.py:154} INFO - [2025-02-17T14:59:46.715+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T14:59:46.736+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.053 seconds
[2025-02-17T15:00:17.841+0000] {processor.py:157} INFO - Started process (PID=430) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:00:17.842+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T15:00:17.843+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:00:17.843+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:00:18.760+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:00:18.787+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:00:18.786+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T15:00:18.812+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:00:18.812+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T15:00:18.838+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.000 seconds
[2025-02-17T15:00:49.902+0000] {processor.py:157} INFO - Started process (PID=436) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:00:49.903+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T15:00:49.905+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:00:49.904+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:00:50.832+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:00:50.860+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:00:50.859+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T15:00:50.888+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:00:50.887+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T15:00:50.912+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.014 seconds
[2025-02-17T15:01:22.098+0000] {processor.py:157} INFO - Started process (PID=442) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:01:22.099+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T15:01:22.101+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:01:22.100+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:01:23.009+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:01:23.037+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:01:23.036+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T15:01:23.065+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:01:23.065+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T15:01:23.091+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 0.996 seconds
[2025-02-17T15:01:53.308+0000] {processor.py:157} INFO - Started process (PID=448) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:01:53.310+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T15:01:53.311+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:01:53.311+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:01:54.234+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:01:54.260+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:01:54.260+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T15:01:54.285+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:01:54.284+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T15:01:54.311+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.006 seconds
[2025-02-17T15:02:25.494+0000] {processor.py:157} INFO - Started process (PID=454) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:02:25.495+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T15:02:25.496+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:02:25.496+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:02:26.441+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:02:26.469+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:02:26.468+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T15:02:26.498+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:02:26.498+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T15:02:26.520+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.030 seconds
[2025-02-17T15:02:57.626+0000] {processor.py:157} INFO - Started process (PID=460) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:02:57.627+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T15:02:57.629+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:02:57.628+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:02:58.614+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:02:58.642+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:02:58.641+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T15:02:58.671+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:02:58.671+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T15:02:58.699+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.076 seconds
[2025-02-17T15:03:29.709+0000] {processor.py:157} INFO - Started process (PID=466) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:03:29.711+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T15:03:29.712+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:03:29.712+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:03:30.882+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:03:30.915+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:03:30.914+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T15:03:30.944+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:03:30.944+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T15:03:30.972+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.268 seconds
[2025-02-17T15:04:01.113+0000] {processor.py:157} INFO - Started process (PID=472) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:04:01.115+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T15:04:01.116+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:04:01.116+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:04:02.258+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:04:02.289+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:04:02.288+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T15:04:02.317+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:04:02.317+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T15:04:02.344+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.236 seconds
[2025-02-17T15:04:33.322+0000] {processor.py:157} INFO - Started process (PID=478) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:04:33.324+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T15:04:33.325+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:04:33.325+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:04:34.398+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:04:34.434+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:04:34.433+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T15:04:34.462+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:04:34.462+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T15:04:34.491+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.173 seconds
[2025-02-17T15:05:04.604+0000] {processor.py:157} INFO - Started process (PID=484) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:05:04.606+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T15:05:04.607+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:05:04.606+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:05:05.561+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:05:05.585+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:05:05.584+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T15:05:05.613+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:05:05.613+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T15:05:05.668+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.068 seconds
[2025-02-17T15:57:09.275+0000] {processor.py:157} INFO - Started process (PID=496) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:57:09.288+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T15:57:09.298+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:57:09.297+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:57:16.244+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:57:16.542+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:57:16.540+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T15:57:16.692+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:57:16.684+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T15:57:16.774+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 7.516 seconds
[2025-02-17T15:57:47.530+0000] {processor.py:157} INFO - Started process (PID=502) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:57:47.532+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T15:57:47.535+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:57:47.534+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:57:48.471+0000] {processor.py:839} INFO - DAG(s) dict_keys(['ETL_Pipeline']) retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T15:57:48.498+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:57:48.497+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-02-17T15:57:48.524+0000] {logging_mixin.py:154} INFO - [2025-02-17T15:57:48.524+0000] {dag.py:3722} INFO - Setting next_dagrun for ETL_Pipeline to 2025-02-17T00:00:00+00:00, run_after=2025-02-18T00:00:00+00:00
[2025-02-17T15:57:48.546+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 1.027 seconds
[2025-02-17T16:19:20.903+0000] {processor.py:157} INFO - Started process (PID=173) to work on /opt/airflow/dags/etl_pipeline.py
[2025-02-17T16:19:20.907+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_pipeline.py for tasks to queue
[2025-02-17T16:19:20.912+0000] {logging_mixin.py:154} INFO - [2025-02-17T16:19:20.911+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T16:19:25.417+0000] {logging_mixin.py:154} INFO - [2025-02-17T16:19:25.394+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline.py", line 13, in <module>
    from src.etl.extract_mongodb import get_validated_mongodb_data
  File "/opt/airflow/src/etl/extract_mongodb.py", line 4, in <module>
    from pymongo.mongo_client import MongoClient
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T16:19:25.426+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline.py
[2025-02-17T16:19:25.462+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_pipeline.py took 4.569 seconds
